### PR TITLE
[ES-558] Added websub properties

### PR DESCRIPTION
**Pull request by RitikJain4108**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: id-authentication-default.properties [hunk 1]
```diff
@@ -271,6 +271,7 @@ ida-topic-fraud-analysis=IDA_FRAUD_ANALYTICS
 ida-topic-auth-anonymous-profile=ANONYMOUS_PROFILE
 ida-topic-pmp-oidc-client-created=OIDC_CLIENT_CREATED
 ida-topic-pmp-oidc-client-updated=OIDC_CLIENT_UPDATED
+ida-topic-remove-id-status=REMOVE_ID_STATUS
```
Comment (mahammedtaheer) [inline id-authentication-default.properties:274]: Where is the websub secret configured for this topic?

Comment (RitikJain4108) [reply id-authentication-default.properties:274]: We will create a role for 'mpartner-default-auth' in keycloak to publish this topic, there is no secrets created.

Comment (RitikJain4108) [reply id-authentication-default.properties:274]: Secret will be used for subscription in idrepo that I added in idrepo config.

Comment (mahammedtaheer) [reply id-authentication-default.properties:274]: secret is not required in case of event publishing to websub, it needs only authentication.

**Review by mahammedtaheer — APPROVED** [review]

**Pull request merged**